File: pix/src/main/java/com/fxn/utility/RegexUtil.java
package com.fxn.utility;

/**
 * Created by sangcomz on 09/04/2017.
 */

public class RegexUtil {
    private static final String GIF_PATTERN = "(.+?)\\.gif$";

    public boolean checkGif(String path) {
        return path.matches(GIF_PATTERN);
    }
}

File: pix/src/main/java/com/fxn/utility/RegexUtil.kt
package com.fxn.utility

/**
 * Created by sangcomz on 09/04/2017.
 */
class RegexUtil {
    fun checkGif(path: String): Boolean {
        return path.matches(GIF_PATTERN.toRegex())
    }

    companion object {
        private const val GIF_PATTERN = "(.+?)\\.gif$"
    }
}